### DISCUSSION TITLE
The usage of transformation in tutorial is not correct

### DISCUSSION THREAD

**Discussion by Jingnan-Jia** [Project-MONAI/MONAI#1476] [Q&A] [answered]
In this [3D segmentation tutorial](https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/challenge_baseline/run_net.py), two transformations are used:

```
                    rotate_range=(-0.05, 0.05),
                    scale_range=(-0.1, 0.1),
```

I think the two lines make the usage confusing. The two lines implicit the usage of `rotate_range` and `scale_range` should be 
```
                    rotate_range=(min, max),
                    scale_range=(min, max),
```

It made me thought that the `rotation_range` for the 3 dimentions would be the same: a random value between (-0.05, 0.05) , and I also thought that the `scale_range` for the 3 dimentions would be the same: a random value between (-0.1, 0.1) . 

However, it is not the truth. Actually, the `rotation_range` for the first dimention would be a random value between (0.05, -0.05) , the `rotation_range` for the second dimention would be another random value between (-0.05, 0.05) , and the `rotation_range` for the third dimention would be `None`. Because in the [source code](https://github.com/Project-MONAI/MONAI/blob/master/monai/transforms/spatial/array.py):

```
            self.rotate_params = [self.R.uniform(-f, f) for f in self.rotate_range if f is not None]
```

So the correct usage should be
```
                    rotate_range=(rotate_range_1, rotate_range_2, rotate_range_3),
                    scale_range=(scale_range_1, scale_range_2, scale_range_3),
```
**And it is not necessary to set `rotate_range_1` a negative value at all!**

Therefore I advice to replace the original code from 
```
                    rotate_range=(-0.05, 0.05),
                    scale_range=(-0.1, 0.1),
```
to
```
                    rotate_range=(0.05, 0.05, 0.5),
                    scale_range=(0.1, 0.1, 0.1),
```
so that we can tell the readers the correct usage of the `rotate_range` and `scale_range`.

**Comment by rijobro** [comment, marked as answer]
Hi, good spot that certainly looks like a typo to me! Would you be interested in creating a PR to address this issue? 

Note that your suggested changes would change the output slightly, since previously the 3rd dimension would have been `None`. If you find the results to be degraded (which I doubt), then maybe switch to using `rotate_range=(0.05, 0.05, None)`, etc.

**Reply by Jingnan-Jia** [reply]
I have already created a [PR](https://github.com/Project-MONAI/tutorials/pull/111), please help to review it.
